ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/utils/TestUtils.java

```diff
@@ -72,6 +75,12 @@
     return priorCommands;
   }
 
+  private static KsqlEngineMetrics buildEngineMetricsMock() {
```

Comment:
Mockito!

Reply:
Will do once I merge to master - no Mockito in 5.1.